.github/workflows/ci.yaml

```diff
@@ -111,11 +111,14 @@ jobs:
       - name: Install Rosetta 2
         run: echo "A" | softwareupdate --install-rosetta || true 
       - run: brew install go lz4 automake autoconf libtool 
+        shell: zsh {0}
```

Comment:
What is the significance of adding this line?

Reply:
https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstepsshell

Since macOS 10.15.x, the default shell on macOS has been `zsh`. The default shell for GitHub workflows is `bash`. As I noted in the PR description, Homebrew install configures ZSH specific environment because ZSH is the default shell. Since ARM macOS runners do not have `brew` installed to a "default" path (e.g. `/usr/bin`) and instead have it installed to `/opt/homebrew`, if running bash instead of a configured ZSH, `brew` will not be found.